src/pages/blog/20221202-meetcommerce.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: '@/layouts/post.astro'
+date: 2022-12-02
+title: Meet you soon in Madrid!
+description: Meet Commerce Spain is taking place on Tuesday 13th December, in Madrid, Spain, an Mage-OS will meet you there!
+---
+
+![A banner of the Mage-OS talk Vinai Kopp will be giving at MeetCommerce.es.](/blog/2022/2022-12-02-meetcommerce-mage-os-talk-bannner.jpg)
+
+[Meet Commerce Spain](https://meetcommerce.es/) is taking place on Tuesday 13th December, in Madrid, Spain.
+
+For a couple of months, the team at Mage-OS has been preparing for this event with much enthusiasm and excitement.
+
+Why? Because it felt so good to catch up in person at Meet Magento New York 2022, we wanted to do it again. Also, this event is strongly connected with Mage-OS as Ignacio Riesco is CEO of the organizing agency Interactiv4, and also sits on the board of Mage-OS.
+
+## Mage-OS at Meet Commerce
+### Mage-OS Presentation
+
+Vinai Kopp, Board President and founding member of Mage-OS, will be presenting at Meet Commerce Spain: [“Mage-OS: Fork it, Own it, Grow it, Better”](https://www.linkedin.com/feed/update/urn:li:activity:6996423767186923520).
+
+This talk is inspired by all the conversations we've had with you, people and friends from the Magento Community, who have been expressing support, giving constructive feedback, and asking questions ([lots of questions](https://mage-os.org/frequently-asked-questions)).
+
+The talk will cover Mage-OS’s vision and goals, purpose, plans and progress. You won't want to miss it!
+
+Fun fact: Even the presentation title came up from Discord casual conversation with our community!
+
+### Mage-OS Booth
+As a Silver Sponsor, we’ll have a nice booth at the conference. Come find us to chat Mage-OS, hang out, and get your Mage-OS button badge!
+
+Here is a list of Mage-OS board members and contributors who are confirmed at this stage. Some also have talks in their own fields of expertise (see the agenda).
+- Vinai Kopp
+- Ignacio Riesco
+- John Hughes
+- Maciej Kalkowski
+- Peter Jaap Blaakmeer
+- Willem Wigman
+
+As a nice thank-you gift, Interactiv4 has offered free tickets to individuals who are contributing to Mage-OS efforts, whether on the technical, content, or organizational side. If this is you and you're interested in attending, let us know!
+
+[Join our Mage-OS Discord](https://discord.gg/9xHBTTpa) to find and connect with other attendees!
+
+![A photo of happy Magento people being happy about the happy organization Mage-OS is.](/blog/2022/2022-12-02-meetcommerce-mageos-2.jpg)
+
+## Meet Commerce Madrid: what else is different?
```

Comment:
```suggestion
## Meet Commerce Spain: what else is different?
```
Need to update to 'Spain' as per the Google Doc for consistency. Otherwise, it all looks good to me :)